FirebaseAuth/Sources/Public/FirebaseAuth/FIRTOTPMultiFactorAssertion.h

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Please add the new public APIs to ./FirebaseAuth/Tests/Sample/SwiftApiTests/SwiftAPI.swift to confirm they match the new approved public APIs

Reply:
Does this solve the `New public headers were added, did you remember to add them to the umbrella header` warning?  If not, can you suggest how do we verify this is addressed? (I saw the FirebaseAuth_umbrella.h in `Support Files` in my dev env has already reflected with the new public headers).

Reply:
That warning is unrelated to addressing API test coverage. Where are you seeing the public header warning?

Reply:
> 1 Warning
> ⚠️	New public headers were added, did you remember to add them to the umbrella header?
> Generated by 🚫 [Danger](https://danger.systems/)

This is generated with this PR on May 9th. It is still attached to this PR.

Reply:
Hmm, the FirebaseAuth.h changes should mitigate this warning. 

It looks like there's something wrong in the Dangerfile https://github.com/firebase/firebase-ios-sdk/blob/master/Dangerfile#L156.

@morganchen12 Any ideas?